index.js

```diff
@@ -10,7 +11,7 @@ app.get('/rebels', (request, response) => {
 })
 
 app.get('/rebels/:identifier', (request, response) => {
-  if (typeof request.params.identifier === 'integer') {
+  if (!isNaN(request.params.identifier)) {
```

Comment:
Nice job. You also could have handled this with an OR in Sequelize like we did in the NFL and SuperHeroes APIs

```JavaScript
    const rebels = await models.Rebels.findAll({
        where: {
            [Op.or]: [{ id: identifier }, { callSign: identifier }]
        }
    })
```